### ISSUE TITLE
Accessibility - Each form control must have associated label

### ISSUE THREAD

**Issue by tworkman512** [codemirror/CodeMirror#4371]
Can someone provide some guidance on how to add a <label/> for a CodeMirror <textarea/> so that I can make form control pass for accessibility? Seems like this should be a simple thing to achieve but haven't found anything helpful yet. Thanks!

**Comment by marijnh** [comment]
Does calling `getInputField()` and setting attributes on the resulting DOM element help?

**Comment by adrianheine** [comment]
I'm closing this for now since there was no further feedback. CodeMirror in general is currently not very accessible, though.

However, we are working on a rewrite (CodeMirror 6) that will greatly improve accessibility, and we are currently raising money for this work: See [the announcement](https://codemirror.net/6/) for more information about the rewrite and a demo.

Note that CodeMirror 6 is by no means stable or usable in production, yet. It is highly unlikely that we pick up this issue for CodeMirror 5, though.
